src/parser/parser.h

```diff
@@ -322,6 +324,7 @@ class CVC4_PUBLIC Parser {
    *
    * @param name the name of the variable
    * @return the variable expression
+   * Only returns a variable if its name is not overloaded.
    */
   Expr getVariable(const std::string& name);
```

Comment:
Also, this needs a better name.  We have getVariableExpressoin and getVariable - the method names should distinguish how these are different - maybe this one should be lookupVariable?